src/App.tsx

```diff
@@ -1,26 +1,25 @@
 import './App.css';
-import { ChangeEvent, FormEvent, useEffect, useRef, useState } from 'react';
-import { addTask, deleteSingleTask, getTasks, updateTask } from './utils/api';
-import { Task } from './utils/types';
+import { ChangeEvent, FormEvent, useRef, useState } from 'react';
 import TaskList from './components/TaskList/TaskList';
 import DeleteModal from './components/DeleteModal/DeleteModal';
+import useTaskManager from './hooks/useTaskManager';
```

Comment:
Styling 💅: don't use a default export for hooks, use a named export.

Reply:
Updated to named export